01_host/03_transition/02_extrinsics.adoc

```diff
@@ -161,8 +161,8 @@ be included in the current block as explained in <<algo-build-block>>.
 |Unix epoch time (<<defn-unix-time>>)
 
 |uncles00
-|Array of block headers
-|Provides a list of potential uncle block headers (<<defn-block-header>>) for a given block
+|Array of block headers (auxiliary information)
+|Provides a list of potential uncle block headers (<<defn-block-header>>) for a given block, if any.
```

Comment:
Uncles are not used anymore and I think they never were. Meaning you don't need to pass the uncles.